src/components/DispositionView.js

```diff
@@ -451,12 +452,12 @@ class DispositionView extends React.Component {
 
         {
           !isInReview()
-          ? <FormNav
-              prevPath={FORM_PATHS.COMPLAINANT}
-              nextPath={FORM_PATHS.OFFICER}
-              handlePageChange={this.handlePageChange}
-              setDidClickNav={this.setDidClickNav} />
-          : null
+            ? <FormNav
```

Comment:
A note on style, I like doing this because I think it looks cleaner:

```
{
  !isInReview()
    ? (
      <FormNav
          ... />
    )
    : null
}
```

Reply:
Updated